data_model/src/isi.rs

```diff
@@ -181,22 +181,30 @@ pub struct Pair {
 }
 
 /// Composite instruction for a sequence of instructions.
-#[derive(
-    Debug, Display, Clone, PartialEq, Eq, Decode, Encode, Deserialize, Serialize, IntoSchema,
-)]
-#[display(fmt = "{instructions:?}")] // TODO: map to Display.
+#[derive(Debug, Clone, PartialEq, Eq, Decode, Encode, Deserialize, Serialize, IntoSchema)]
 pub struct SequenceBox {
     /// Sequence of Iroha Special Instructions to execute.
     pub instructions: Vec<Instruction>,
 }
 
+impl core::fmt::Display for SequenceBox {
+    fn fmt(&self, f: &mut core::fmt::Formatter<'_>) -> core::fmt::Result {
+        write!(f, "SEQUENCE [")?;
```

Comment:
Would be grate to reuse `format_comma_separated`, can't we place it in common crate for `core` and `data_model`?

Reply:
Oh, I could do that, but that sounds more like for `utils` crate rather than `data_model`. Well, let's keep in `data_model` for now